src/components/Tabs/Tabs.mdx

```diff
@@ -120,32 +130,26 @@ const animalCrossingData = [
 
 ## Responsive Mobile Carousel
 
-For both, the `tabsData` prop approach and the children component approach
-(described below in this document), the mobile carousel feature is built. This
-adds "previous" and "next" arrows in the tab list for users to scroll through
-all the tabs while the panel stays static.
-
-Note that for the children component approach, we expect a `TabList` component
-to be passed or else the mobile carousel will _not_ be built. More on this in
-the <Link href="#children-components" target="_self">Children Components</Link> section.
+- For mobile viewports, the `Tabs` component will generate a carousel feature.
+- The carousel feature adds `"previous"` and `"next"` buttons that can be used to scroll through
+  all tabs while the panel stays static.
 
 To view and test this in Storybook, go to the "Canvas" tab and change the
 viewport in the Storybook toolbar at the top of the page.
 
-## Scrollable Desktop Tab Set
-
-For desktop viewports (600px and greater), the full tab set scrolls left/right when
-the width of the tab set is greater than the width of the viewport.
+## Horizontal Scrolling
 
-To scroll horizontally, use the `shift + scroll wheel` combination.
+- When the width of the full tab set is greater than the available horizontal
+  space, the overflow will be hidden and horizontal scrolling will be enabled.
+- This functionality is the same for desktop, tablet, and mobile.
+- Horizontal scrolling can be accomplished by using the trackpad, the `shift + scroll wheel` combination, or the keyboard to tab through the available tabs.
```

Comment:
Line 142 is broken into two but this line is not.

Reply:
I use a plugin to automate the text wrap at 80 chars, but that plugin split up the text in the back tics. I left it alone while I was working, but I meant to go back to add the line breaks manually. Oops. I just pushed an update with the wrapping cleaned up.